src/modules/roc_fec/writer.cpp

```diff
@@ -256,6 +277,21 @@ void Writer::encode_repair_packets_() {
     encoder_.fill();
 }
 
+void Writer::compose_repair_packets_() {
+    for (packet::seqnum_t i = 0; i < cur_rblen_; i++) {
```

Comment:
I guess that we can use `size_t` instead of `packet::seqnum_t`.

Reply:
Done.